docs/Charting-Concepts/AreaChart.md

```diff
@@ -0,0 +1,131 @@
+# Contributor guide: Area Chart
+
+![Areachartpic1.png](images/Areachartpic1.png)
```

Comment:
Could you add a section in https://github.com/microsoft/fluentui-charting-contrib/blob/main/apps/docsite/sidebars.ts

{
      type:'category',
      label:'Charting Concepts',
      items:[
        {
          type: 'autogenerated',
          dirName:'Charting-Concepts',
        }
      ]
    },

Reply:
Done